devimint/src/tests.rs

```diff
@@ -1812,6 +1813,142 @@ pub async fn recoverytool_test(dev_fed: DevFed) -> Result<()> {
     Ok(())
 }
 
+pub async fn guardian_backup_test(dev_fed: DevFed, process_mgr: &ProcessManager) -> Result<()> {
+    use fedimint_core::core::LEGACY_HARDCODED_INSTANCE_ID_WALLET;
+
+    log_binary_versions().await?;
+
+    let fedimint_cli_version = crate::util::FedimintCli::version_or_default().await;
+    let fedimintd_version = crate::util::FedimintdCmd::version_or_default().await;
+    if VersionReq::parse("<0.3.0-alpha")?.matches(&fedimint_cli_version)
+        || VersionReq::parse("<0.3.0-alpha")?.matches(&fedimintd_version)
```

Comment:
Good candidate to tag with

```
// TODO(support:v0.2)
```
https://github.com/fedimint/fedimint/issues/4424

Reply:
Will do a follow-up for your nits :)